designs/penelope/src/shape.mjs

```diff
@@ -34,15 +38,14 @@ export const options = {
 }
 
 export function BuildMainShape(part, frontPart) {
-  const { sa, options, measurements, Point, Path, points, paths, store, paperless, macro } =
+  const { sa, options, measurements, optionalMeasurements, Point, Path, points, paths, store, paperless, macro } =
     part.shorthand()
 
   let skirtLength = measurements.waistToKnee * (1 + options.lengthBonus) // + options.hem;
 
   store.set('skirtLength', skirtLength)
   store.set('waistEase', measurements.waist * options.waistEase)
   store.set('seatEase', measurements.seat * options.seatEase)
-  // [joost] I don't like how hem is handled here rather than as hem allowance
```

Comment:
I do not understand what Joost's concerns were. The original code appears fine to me, where there is a variable hem size that scales with the waist-to-knee measurement.

(You can mark this comment as Resolved.)